druid/examples/image.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! This example shows how to draw an SVG.
+//!
+//! Requires the non-default "svg" feature to be enabled:
+//! `cargo run --example svg --features "svg"`
+//!
+
+#[cfg(not(feature = "images"))]
+fn main() {
+    eprintln!("This examples requires the \"images\" feature to be enabled:");
```

Comment:
just for fun I'll point out that you can escape newlines in strings with `\`, so you could write this as,

```rust
    eprintln!(
        "This examples requires the \"images\" feature to be enabled:\n\
          cargo run --example image --features \"images\""
    );
```

Is this any better? probably not. 🤷‍♂

Reply:
Again, i just copied from svg. shall I tweak both?

Reply:
na, I don't think my version is much improvement.